### PR TITLE
WIP: migrate away from jQuery

### PR DESCRIPTION
**Pull request by DanielRuf**
## Description

...

- Closes 

## Types of changes

- [ ] Documentation
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (anything that would change an existing functionality)
- [ ] Maintenance (refactor, code cleaning, development tools...)

## Checklist

- [ ] I have read and follow the CONTRIBUTING.md document.
- [ ] The pull request title and template are correctly filled.
- [ ] The pull request targets the right branch (`develop` or `develop-v...`).
- [ ] My commits are correctly titled and contain all relevant information.
- [ ] I have updated the documentation accordingly to my changes (if relevant).
- [ ] I have added tests to cover my changes (if relevant).

### CODE REVIEW

**Comment by axten** [comment]
would be great to have this in the near future!
Currently our projects don't use the JS Components because of not having jquery...

**Comment by rafibomb** [comment]
Super cool! More of an F7 thing right?

**Comment by DanielRuf** [comment]
> Super cool! More of an F7 thing right?

Exactly.

**Comment by kball** [comment]
@DanielRuf what is the status of this PR/branch. Do you want to keep working on this or should we merge this as valid progress?

**Comment by DanielRuf** [comment]
@kball this is still WIP and I have to continue this one. Can not be merged, especially without writing more tests.

File: js/foundation.abide.js [hunk 1]
```diff
@@ -696,7 +696,7 @@ Abide.defaults = {
    */
   validators: {
     equalTo: function (el, required, parent) {
-      return $(`#${el.attr('data-equalto')}`).val() === el.val();
+      return $(`#${el.attr('data-equalto')}`).get(0).value === el.get(0).value;
```
Comment (rediris) [inline js/foundation.abide.js]: Should use `el.getAttribute()` instead of `el.attr`

Comment (DanielRuf) [reply js/foundation.abide.js]: Please check the title of this PR, it's WIP ;-)

**Pull request closed without merging**